src/main/java/ca/team2706/frc/robot/subsystems/DriveBase.java

```diff
@@ -410,7 +493,9 @@ public double getRightSpeed() {
      * @return The rotation of the robot
      */
     public double getHeading() {
-        return gyro.getFusedHeading();
+        double[] yawPitchRoll = new double[3];
```

Comment:
Mentioned already, but a method for getting the robot's yaw would be helpful for everyone.